components/mdx/mermaid.tsx

```
@@ -0,0 +1,219 @@
+      dangerouslySetInnerHTML={{ __html: svg }}
+    />
+  );
+
+  if (isFullscreen) {
+    return (
+      <div className="fixed inset-0 z-50 flex items-center justify-center bg-fd-background/95 backdrop-blur-sm">
+        <button
+          onClick={() => setIsFullscreen(false)}
+          className="absolute top-4 right-4 p-2 rounded-lg bg-fd-muted hover:bg-fd-accent transition-colors z-10"
+          aria-label="Close fullscreen"
+        >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)